lib/plugins/python/index.js

```diff
@@ -1,16 +1,45 @@
 import { PYTHON_IMPORT } from '../../../packages/helper-grammar-regex-collection/index.js';
 import insertLink from '../../insert-link';
 import preset from '../../pattern-preset';
+import liveResolverQuery from '../../resolver/live-resolver-query.js';
+import relativeFile from '../../resolver/relative-file.js';
 
 export default class Python {
 
+  static clickHandler({ path, target }) {
```

Comment:
If you agree with my comment at https://github.com/OctoLinker/browser-extension/pull/274/files#r99658736, we should also change `clickHandler` to `resolve` here.